gdpr/impl.go

```diff
@@ -18,7 +19,7 @@ import (
 type permissionsImpl struct {
 	cfg             config.GDPR
 	vendorIDs       map[openrtb_ext.BidderName]uint16
-	fetchVendorList func(ctx context.Context, id uint16) (vendorlist.VendorList, error)
+	fetchVendorList []func(ctx context.Context, id uint16) (vendorlist.VendorList, error)
```

Comment:
Probably best to make this a map. Otherwise it will start to break down when TCF 1 support is removed.